src/Components/Navigation.js

```diff
@@ -168,7 +168,7 @@ export const Navigation = Scrivito.connect(
             <NavigationSection heightClassName={heightClassName} />
             <ScrollToNextSectionLink heightClassName={heightClassName} />
           </Scrivito.BackgroundImageTag>
-          <div id="mainContent" />
+          <div id="nextSection" />
```

Comment:
If I remember correctly we introduced this `div` back in the days, when we used a dedicated React library to to the scrolling. Since https://github.com/Scrivito/scrivito_example_app_js/commit/d9942c965bf8d0451e65e81b90aa7f4ce0b1a4ca `div` here is strictly speaking no longer needed.

How about the following:
* Wrap `<Scrivito.CurrentPage />` into `<div id="mainContent">` as you did in this PR.
* Remove this line.
* Leave `ScrollToNextSectionLink` to link to `#mainContent`.

This way we only have one div to jump to and the ID stays the same.

The only downside I see, is that `ScrollToNextSectionLink` now relies on `<App>` to render a `mainContent` id. But it previously relied on `<Navigation>` to do just that.